src/controllers/server.js

```diff
@@ -221,7 +223,7 @@ class Server extends EventEmitter {
             if (err) {
                 if (err.statusCode === 404) { // container not found
                     this.log.error('The container for this server could not be found. Trying to rebuild it.');
-                    this.emit('console', `${Ansi.style.red}(Daemon) The container of this server could not be found and has to be rebuilt.`);
+                    this.emit('console', `${Ansi.style.cyan}(Daemon) The container of this server could not be found and needs to be rebuilt.`);
```

Comment:
I'd say just completely remove this since the next line is one saying the container is being rebuilt.